packages/utilities/src/EventGroup.ts

```diff
@@ -71,7 +72,7 @@ export class EventGroup {
     eventName: string,
     eventArgs?: any,
     bubbleEvent?: boolean
-  ) {
+  ): any {
```

Comment:
Should be able to assign the type to this.

Reply:
boolean | undefined?